source/collections/LinkedList.ooc

```diff
@@ -0,0 +1,155 @@
+import structs/List
+
+LinkedList: class <T> {
+	_size = 0 : SizeT
+	size ::= _size as Int
+	head: Node<T>
+	init: func {
+		head = Node<T> new()
```

Comment:
This file is missing a looot of `this`. I guess it could be fixed later, but if the point of the PR is to clean up, why not?